tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/superhero/model/SuperHeroError.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.graphql.tck.apps.superhero.model;
+
+public class SuperHeroError implements SuperHeroOrError {
```

Comment:
I'm not sure about combining an entity, "SuperHero" with an exception "SuperHeroError".  I think we should handle errors using the GraphQL error handling semantics.  Maybe for this test we could use an interface like `Person` that `SuperHero` could implement.  Then add a new class like `Sidekick` or `Witness` or something - to indicate that they are a person, but not a super hero.  That could include characters like Lois Lane, or Pepper Potts who add value to the super heroes but are not super themselves.  That way they are another entity that could logically be combined with the SuperHero entity.  WDYT?

Reply:
I tried to implement a pattern I saw at the GraphQL conf 2019 where business errors are part of the business object's response and low-level/non-business ones are in the graphql "error" response field.

But I don't mind doing it like you propose, since the goal for TCK is not to start a debate on good/anti pattern !